### DISCUSSION TITLE
Tyra Discussions!

### DISCUSSION THREAD

**Discussion by h4570** [h4570/tyra#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Foxar** [comment]
Hello everyone!
I'm Foxar, im a C++ enthusiast who recently decided to join the Open Source world and start contributing to open source projects. Tyra happened to be one that caught my attention, as it's related to one of my interests, game development. At the time im focused on developing the dolphin sample for the 2020 December release, after which I most likely will start focusong on issues pertaining to Tyra directly.

**Comment by Foxar** [comment]
I thought asking here would be most appropriate.
What should we work on in regards to Tyra as a prority right now? There's some issues that only @h4570 can fix, such as those within the rendering (which I doubt anyone else can realistically fix), and there's also some issues that don't seem very important (such as the boundingbox idea #72  for example.)
We would benefit from some kind of priority sorting of issues. Maybe new labels?

**Comment by Br4k2n** [comment]


**Reply by h4570** [reply]
Hi @TRB46 
Current Tyra version is pretty outdated.
However from couple of months we are working strongly at Tyrav2, which initial version should be available in 2-3 weeks.
Tyrav2 is based on docker, so it is very easy to use it on Windows - instruction will be included.

**Reply by Br4k2n** [reply]
Oh, Thanks for the reply! I will wait eagerly for Tyrav2!

**Reply by h4570** [reply]
Done!

**Comment by Wellinator** [comment]
Hey guys! I'm Wellington, I'm a C++ enthusiast as well! I usually program using JS/TS and Python, but low level is kinda hobby xD;
I really like to undestando how things works and Tyra helped me a lot this way. I started creating the Cube sample.

**Reply by Wolf3s** [reply]
> Hey guys! I'm Wellington, I'm a C++ enthusiast as well! I usually program using JS/TS and Python, but low level is kinda hobby xD; I really like to undestando how things works and Tyra helped me a lot this way. I started creating the Cube sample.

Well i think this knowledge of phyton will be pretty usefull soon.

**Comment by Wellinator** [comment]
I'm currently working on my own PS2 game using Tyra Engine, [TyraCraft](https://github.com/Wellinator/tyracraft)!
You can take a look and contribute if you wanna :D

**Comment by RoseEternal** [comment]
Sorry to dig up some shallow grave of a project; however I am having an issue with going through the set-up process for Tyra. 

At the part where we can check tutorial code, I assume if everything has been set up correctly we shouldn't have errors anywhere. That's not the case for me. I am having an error on line 11 in "tutorial_04.cpp" for 04-de_dust2. 
Problem 1: `#include errors detected based on information provided by the configurationProvider setting. Squiggles are disabled for this translation unit (C:\Users\Username\Desktop\PS2 Development Kit\Repositories\tyra\tutorials\04-de_dust2\src\tutorial_04.cpp).C/C++(1696)
`
Problem 2: `cannot open source file "stdlib.h" (dependency of "tyra")C/C++(1696)`

If anyone is following this discussion, I'd appreciate the help. Thank you all in advance!

**Reply by Wellinator** [reply]
Have you copied the intellisense folder correctly?

**Reply by RoseEternal** [reply]
Yes, I have. Copied everything in the zip and placed it in the exact folder name "tyra-intellisense" 
![image](https://user-images.githubusercontent.com/62110583/213806156-f2a6125b-d4ca-47fe-8353-282de52a6389.png)

**Comment by russslllan** [comment]
Hello! Sorry off topic question, I'm trying to understand how work bloom on ps2, I've read the presentation "Practical Implementation of SH Lighting and HDR Rendering on PlayStation 2" and still don't understand, Gaussian blur is calculated on VU1 or on GS? If it calculated on VU1, but how do transfer image from Framebuffer to VU1 memory?

**Comment by gamefal1820** [comment]
I was setting up the environment like the first tutorial video, when i running build task, it occurs with this error: `Unexpected parameter Station\repositories\tyra\tutorials\0-4-de_dust2/bin/tutorials_04.elf`.

I've tried to run tutorials_04.elf in PCSX2, it's worked but get stuck in tyra watermark.
repositories path: `D:\My Content Station\repositories`

Can someone help?

**Comment by HolandonHdunto** [comment]
Is there any possibility of using blender 2.79.x to create games for TYRA, as they do with BGE, blende has an interesting system of nodes that can be customized, the logic can be done using nodes like in the Unreal Engine, the materials of the blender can be converted into TYRA compatible materials, the blender would be like a visual interface for TYRA and would not need to create from scratch.

**Comment by C4io-z** [comment]
Hello! I need help whit this issue:
The term './run.ps1' is not recognized as the name of an operable cmdlet, function, script file, or program. Check
the spelling of the name, or if a path was added, verify that the path is correct and try again.
    + CategoryInfo : ObjectNotFound: (./run.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
    + PSComputerName : localhost

**Reply by oGabrielRibeiro** [reply]
Hi, i solved this problem running that command in PowerShell with ADM privileges.

> `Set-ExecutionPolicy Unrestricted`

**Comment by MichaeIDeSanta** [comment]
weird question. but i see dust 2 cs 1.6 map on the tyra page, i was wondering. whats that game called? like, tyra1.6? countertyra? im not sure but i want it
